recipes/flac/CMakeLists.txt

```diff
@@ -0,0 +1,175 @@
+cmake_minimum_required (VERSION 3.1.0)
```

Comment:
I could do this, but my work is largely based on work by someone else: https://github.com/soulik/SDL_mixer/blob/master/external/flac-1.3.1/CMakeLists.txt and I don't think it is 'proper' if I submit this as a PR. I have contacted the author of the original if he is willing to do so, or if he is ok if I do it on his behalf.